cli/migrate/migrate.go

```diff
@@ -1850,7 +1851,12 @@ func (m *Migrate) ExportDataDump(tableNames []string) ([]byte, error) {
 	// to support tables starting with capital letters
 	modifiedTableNames := make([]string, len(tableNames))
 	for idx, val := range tableNames {
-		modifiedTableNames[idx] = fmt.Sprintf(`"%s"`, val) 
+		split := strings.Split(val, ".")
+		if len(split) == 2 {
+			modifiedTableNames[idx] = fmt.Sprintf(`%s."%s"`, split[0], split[1])
```

Comment:
2 points here : 

1. I would suggest that we wrap the schema part (`split[0]`) with `"` as well, for good measure :)
2. I was also thinking about the case when the user provides the input with the `"` in place? I'm thinking we should be handling that case too. Prior to the latest release of the CLI (v1.3) I think some users passed in table names like so `\"Schema_Name\".\"Table_Name\"`, so I guess by handling this we would still be supporting such inputs. What do you think?

Reply:
@kolharsam thanks for your comments! 

1. thanks for pointing that out, I didn't miss it on purpose.
2. although not explicitly here, that case is already "being dealt" because either Go or Postgres are removing the duplicates for `"`. I tried all those cases with this implementation and they all work. I might add a couple of tests for those special cases.

Reply:
Re 2: Yup, it's go that does the trick. But as per my observations, you can escape the \" and it'd be part of the input.

Reply:
@kolharsam I meant with the escape. The escape is simply allowing for a duplicate `"` which is, eventually, considered a duplicate and removed.